armi/nuclearDataIO/cccc/isotxs.py

```diff
@@ -191,6 +191,10 @@ class _IsotxsIO(cccc.Stream):
     """
     A semi-abstract stream for reading and writing to a :py:class:`~armi.nuclearDataIO.isotxs.Isotxs`.
 
+    .. impl:: Tool to read and write ISOTXS files.
```

Comment:
If you make the `_Isotxs` class public, you could move this to the actual `readWrite` method instead of the module. Might be more useful that way. 

Seems odd to me to have a public method within a protected class.

Reply:
Yeah, I considered doing this, but that's a bigger change to the API, that affects like 6 files.

I suppose it doesn't break the API to make private things public though...

Reply:
Fixed!